template/ManagedReference.extension.js

```diff
@@ -22,37 +22,117 @@ function replaceIObservableAndTSource(str){
 // compile list of data for input --> o --> output diagrams
 function defineInputsAndOutputs(model){
   operators = [];
-  if (model.children){
+  if (model.oe.hubOrDevice === 'hub'){
+    description = [ model.__global._shared['~/api/OpenEphys.Onix1.MultiDeviceFactory.yml'].summary, 
+                    model.__global._shared['~/api/OpenEphys.Onix1.MultiDeviceFactory.yml'].remarks].join('');
+    const hubChildrenLength = model.__global._shared['~/api/OpenEphys.Onix1.MultiDeviceFactory.yml'].children.length;
+    for (let i = 0; i < hubChildrenLength; i++){
+      input = {};
+      if (model.__global._shared['~/api/OpenEphys.Onix1.MultiDeviceFactory.yml'].children[i].syntax.parameters && model.__global._shared['~/api/OpenEphys.Onix1.MultiDeviceFactory.yml'].children[i].syntax.parameters[0]){
+      //   for (let j = 0; j < childrenLength; j++){
+      //     if(model.children[j]blablablabla){
+      //       input = {
+      //         'specName': replaceIObservableAndTSource(model.__global._shared['~/api/OpenEphys.Onix1.MultiDeviceFactory.yml'].children[i].syntax.parameters[0].type.specName[0].value),
+      //         'name': model.__global._shared['~/api/OpenEphys.Onix1.MultiDeviceFactory.yml'].children[i].syntax.parameters[0].type.name[0].value.replaceAll(/(IObservable<)|(>)/g, ''),
+      //         'description': removeBottomMargin([ model.__global._shared['~/api/OpenEphys.Onix1.MultiDeviceFactory.yml'].children[i].syntax.parameters[0].description, 
+      //                                             model.__global._shared['~/api/OpenEphys.Onix1.MultiDeviceFactory.yml'].children[i].syntax.parameters[0].remarks].join(''))
+      //       };
+      //     }
+      //   }
+      // }
```

Comment:
Do we need to keep these commented lines?

Reply:
Nope. I will remove them.

Reply:
I removed them and also resolved some broken references in the toc due to my removal of the tutorial placeholder files (the lorem ipsum miniscope and rhs2116 pages). Another build action has initiated. Hopefully in two year when the build process completes we can merge if there are no more broken references.